app/views/propertyLinking/chooseEvidence.scala.html

```diff
@@ -51,14 +59,15 @@
                         items = Seq(
                             RadioItem(
                                 content = Text(messages("hasRatesBill.true")),
-                                value = Some("true")
+                                value = Some("true"),
+                                id = Some ("radio-true")
```

Comment:
pls rename to 
```
hasRatesBill-yes
```
and
```
hasRatesBill-no
```

this seems to be the convention

Reply:
done